extensions/multiline.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Multiline messages
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "James Wheare"
+    period: "2019"
+    email: "james@irccloud.com"
+contributors:
+  -
+    name: "jesopo"
+    email: "jess@jesopo.uk"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the unprefixed `multiline-concat` tag or `multiline` CAP/batch names. Instead, implementations SHOULD use the `draft/multiline-concat` tag, `draft/multiline` CAP and `draft/multiline` batch names to be interoperable with other software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed tag name.
+
+## Introduction
+
+This specification adds a new batch type and tag sent by clients and servers to send messages that can exceed the usual byte length limit and that can contain line breaks.
+
+## Motivation
+
+IRC messages have been traditionally limited by the line-based nature of the protocol and the line length limit of 512. To work around these limitations, client implementations split longer messages into multiple lines, and users need to rely on snippet hosting services to send multiple lines without flooding the chat or having their messages interrupted by other users.
+
+Mulitline messages allow for a more coherent experience that avoids the disconnected messages that result from these workarounds.
+
+## Architecture
+
+### Dependencies
+
+This specification uses the [batch][] capability which MUST be negotiated at the same time. It also uses the [message tags][] and [standard replies][] frameworks.
```

Comment:
What if client enabled `batch` before? E.g. `CAP REQ batch` followed by `CAP REQ draft/multiline`

Or after, but still during registration time

Reply:
Dependent caps are a Whole Thing. Here's my take on how they should work:

https://github.com/ircv3/ircv3-specifications/pull/400#issuecomment-579063998

Reply:
I made this match how we phrase it in labeled-response.